sdk/trustedsigning/Azure.Developer.Signing/src/Azure.Developer.Signing.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the Azure.Developer.Signing client library for developing .NET applications with rich experience.</Description>
```

Comment:
We should customize these; they're the metadata for the package that will appear on NuGet.

Reply:
Good catch, thanks.